### PR TITLE
Use `batchConvert.parent.Window.window.close();` rather than `Qt.quit()`

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
@jeetee mind to check whether this is the right thing to do?

### CODE REVIEW

File: batch_convert.qml [hunk 1]
```diff
@@ -24,7 +24,7 @@ MuseScore {
         text: qsTr("This plugin needs MuseScore 3")
         onAccepted: {
             window.visible = false
```
Comment (jeetee) [inline batch_convert.qml:26]: should no longer be required, as the window (now called pluginId) is closed by the next line

File: batch_convert.qml [hunk 2]
```diff
@@ -47,6 +47,7 @@ MuseScore {
     //Window {
     id: window
```
[the remark this answers is not in the document]
Comment (jeetee) [reply batch_convert.qml:48]: also change `window.visible` on L#40 and L#44 to `pluginId.visible`

**Review by jeetee — CHANGES_REQUESTED** [review]
the result of my review comments has not been tested/validated

File: batch_convert.qml (hunk 1 above)
Comment (Jojo-Schmitz) [reply batch_convert.qml]: Should that name be unique, would it conflict with other plugins?

File: batch_convert.qml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply batch_convert.qml]: Done (but picked batchConvert`)`, thanks.

File: batch_convert.qml (hunk 1 above)
Comment (Jojo-Schmitz) [reply batch_convert.qml]: Better save than sorry, so I picked `batchConvert` as the id, thanks.

**Comment by Jojo-Schmitz** [comment]
> the result of my review comments has not been tested/validated

OK, I'll try it out before merging, thanks!

File: batch_convert.qml (hunk 1 above)
Comment (jeetee) [reply batch_convert.qml]: The id of the main plugin object is unique within the plugin scope; it does not conflict with other plugin instantiations, fyi

Comment (Jojo-Schmitz) [reply batch_convert.qml]: OK, thanks, got it

**Pull request merged**